### PR TITLE
Improve release checking & fix update version bug

### PR DESCRIPTION
**Pull request by canack**
Improved release checking & fixed version update error.

### CODE REVIEW

**Review by gormelof — APPROVED** [review]
💯

File: cmd/release.go [hunk 1]
```diff
@@ -67,31 +72,122 @@ func releaseRootCommand(cmd *cobra.Command, _ []string) {
...
+		} else {
+			failedToolTypes = append(failedToolTypes, toolType)
+		}
 	}
 
-	qwm(ExitCodeSuccess, "project passes release criteria")
+	if len(failedToolTypes) == 0 {
+		returnMSG := fmt.Sprintf("project passes release criteria")
+		qwe(ExitCodeSuccess, errors.New(returnMSG))
+	} else {
```
Comment (yeyisan) [inline cmd/release.go:189]: You do not need these changes to determine if the result was successful or not. 
There is a `Status` field on the root, 
```go
	PlaybookResults struct {
		Status  string             `json:"status" bson:"status"`
		SAST    PlaybookTypeDetail `json:"sast" bson:"sast"`
		DAST    PlaybookTypeDetail `json:"dast" bson:"dast"`
		PENTEST PlaybookTypeDetail `json:"pentest" bson:"pentest"`
		IAST    PlaybookTypeDetail `json:"iast" bson:"iast"`
		SCA     PlaybookTypeDetail `json:"sca" bson:"sca"`
		CS      PlaybookTypeDetail `json:"cs" bson:"cs"`
		IAC     PlaybookTypeDetail `json:"iac" bson:"iac"`
	}
```
It holds the overall status.

**Review by yeyisan — CHANGES_REQUESTED** [review]
Looks good!
Only a minor change is requested.

File: cmd/release.go (hunk 1 above)
Comment (canack) [reply cmd/release.go:189]: Actually you right but we can use kdt like this:
`kdt release -p Juiceshop --sast --cs`

With this situation, kdt checks only `sast` and `cs` types. (Not all)
By the way, kdt must check failed types that only this types.

If you use kdt like this:
`kdt release -p Juiceshop` (With not specific types)
Kdt will check all types.
So, we must log all failed types.

**Review by yeyisan — APPROVED** [review]
🚀

**Pull request closed without merging**